waspc/src/Generator/Button.hs

```diff
@@ -39,6 +40,8 @@ generateButtonComponent wasp button = createTemplateFileDraft dstPath srcPath (J
       let (pathInSrc, exportedIdentifier) = Generator.Entity.getImportInfoForAction wasp action
       return $ object [ "importPath" .= buildImportPathFromPathInSrc pathInSrc
                       , "exportedIdentifier" .= exportedIdentifier
+                      , "updateFn" .= WAction._updateFn action
```

Comment:
You could have just used JSON that action transforms into? Meaning you would do "action" = action. But you did not want to depend on that?

Reply:
you are right, I think I also did not think of it. Also, this is actionData so I suppose it was logical to have pieces of action in it. But might have put the whole action as well.